packages/test-utils/src/logger/graphql/queryGraphQL.ts

```diff
@@ -0,0 +1,33 @@
+// TODO [PRO-629] Move this into the "shared" package.
+
+import { fetch } from "undici";
```

Comment:
`undici` is also not declared as a dependency. Please make sure that this package depends on `"undici": "^5.28.4"` (not on the latest v6)